examples/stm32f051r8/src/bin/async.rs

```diff
@@ -0,0 +1,70 @@
+#![no_std]
```

Comment:
could we make this a little more uniform, it seems like we have:

- `<mcu>/src/bin/async.rs` but `src/main.rs`
- but then `<mcu>_async`

could we just have `<mcu>/src/bin/<sync|async>.rs`?

Reply:
what about the main then? leave it empty?